db/migrate/20151231182601_create_loans.rb

```diff
@@ -0,0 +1,27 @@
+class CreateLoans < ActiveRecord::Migration
+  def change
+    create_table :loans do |t|
+      t.references :division, index: true, foreign_key: true
+      t.string :name
+      t.references :primary_agent, references: :people
+      t.references :secondary_agent, references: :people
+      t.string :status
+      t.decimal :amount
+      t.references :currency, index: true, foreign_key: true
+      t.decimal :rate
+      t.integer :length_months
```

Comment:
Possibly, it might depend on whether `target_end_date` ever moves or if it's permanently fixed at the length of the loan. Presumably the number of months the loan term is for is a fixed part of the terms, a target end date could potentially be a timeline that changes as circumstances do, but I'm not sure if that's how it's used. Both fields were in the schema document I was working from.